x-pack/plugins/maps/public/classes/styles/vector/components/data_mapping/ordinal_data_mapping_popover.tsx

```diff
@@ -18,6 +18,7 @@ import {
   EuiText,
   EuiToolTip,
 } from '@elastic/eui';
+import type { _SingleRangeChangeEvent } from '@elastic/eui/src/components/form/range/types';
```

Comment:
Instead of _SingleRangeChangeEvent, would it be possible to use EuiRangeProps['onChange'], like in https://github.com/elastic/kibana/blob/56e45b19c57371599ca0e164bfec1b417f1aa91a/x-pack/plugins/maps/public/classes/sources/es_geo_grid_source/resolution_editor.tsx?

Reply:
Unfortunately not because of the syntax used. `onSigmaChange` is using `function() {}` as opposed to `const someFn = () => {}`, and functions cannot be typed to `EuiRangeProps['onChange']` the same way as `const`s can (at least, I couldn't find a way to accomplish this in my google journeys - I'd love to be wrong on this, please tell me if you know of a way to do so!)

I also attempted to use `Parameters<EuiRangeProps['onChange']>[0]` but unfortunately Typescript couldn't figure that out - I think possibly due to some bizarre combination of reaching into an interface for a key and that key being optional/potentially undefined.

Importing the event type directly was the easiest way I could figure out to update typings without changing your code, but if you're okay with changing your syntax - sure, we can make that change :)